### PR TITLE
Update create-a-phone-system-auto-attendant.md

### PR DESCRIPTION
**Pull request by CLYVR**
Adding information about the supported languages for TTS and Speech Recognition.

### CODE REVIEW

**Comment by CLYVR** [comment]
Updating AA and CQ to include supported langauges.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-a-phone-system-auto-attendant-languages.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+|Slovenian (Slovenia)                    |Y                        |                                       |	
+|Spanish (Mexico)                        |Y                        |Y                                      |
+|Spanish (Spain, International)          |Y                        |Y                                      |
+|Swedish (Sweden)                        |Y                        |                                       |	
+|Thai (Thailand)                         |Y                        |                                       |	
+|Turkish (Turkey)                        |Y                        |                                       |	
+|Vietnamese (Vietnam)                    |Y                        |                                       |	
+
+> [!NOTES]
+> Choosing a language that does not support speech recognition will disable the voice intputs option within the menu and dial by name will be restricted to using keypad (DTMF) entry only. Please refer to [Dial by Name - Keypad (DTMF) entry](dial-voice-reference#dial-by-name---keypad-dtmf-entry) for more information.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant-languages.md:72]:
```suggestion
> Choosing a language that does not support Speech Recognition will disable the voice inputs option within the menu and dial by name will be restricted to using keypad (DTMF) entry only. Please refer to [Dial by Name - Keypad (DTMF) entry](dial-voice-reference#dial-by-name---keypad-dtmf-entry) for more information.
```

File: Teams/create-a-phone-system-auto-attendant-languages.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+>
+> When using text to speech, the text must be entered in the selected language as the system does not perform translation.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant-languages.md:74]:
```suggestion
> When using Text to Speech, the text must be entered in the selected language as the system does not perform translation.
```

File: Teams/create-a-phone-system-auto-attendant.md [hunk 3]
```diff
@@ -52,10 +52,13 @@ To set up an auto attendant, in the Teams admin center, expand **Voice**, click
 
 3. Specify the time zone for this auto attendant. The time zone is used for calculating business hours if you [create a separate call flow for after hours](#call-flow-for-after-hours).
 
-4. Specify a language for this auto attendant. This the language that will be used for system-generated voice prompts.
+4. Specify a [supported language](create-a-phone-system-auto-attendant-languages.md) for this auto attendant. This the language that will be used for system-generated voice prompts.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md:55]:
```suggestion
4. Specify a [supported language](create-a-phone-system-auto-attendant-languages.md) for this auto attendant. This is the language that will be used for system-generated voice prompts.
```

File: Teams/create-a-phone-system-auto-attendant.md [hunk 4]
```diff
@@ -52,10 +52,13 @@ To set up an auto attendant, in the Teams admin center, expand **Voice**, click
...
 
 5. Choose if you want to enable voice inputs. When enabled, the name of every menu option becomes a speech-recognition keyword. For example, callers can say "One" to select the menu option mapped to key 1, or they can say "Sales" to select the menu option named "Sales."
 
+> [!NOTE]
+> If you choose a language in step 4 that doesn't support voice inputs this option will be disabled.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md:60]:
```suggestion
> If you choose a language in Step 4 that doesn't support voice inputs this option will be disabled.
```

File: Teams/create-a-phone-system-call-queue-languages.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+|Slovenian (Slovenia)                    |
+|Spanish (Mexico)                        |
+|Spanish (Spain, International)          |
+|Swedish (Sweden)                        |
+|Thai (Thailand)                         |
+|Turkish (Turkey)                        |
+|Vietnamese (Vietnam)                    |
+
+> [!NOTE]
+> > When using text to speech, the text must be entered in the selected language as the system does not perform translation.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-call-queue-languages.md:72]:
```suggestion
> > When using Text to Speech, the text must be entered in the selected language as the system does not perform translation.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**